### DISCUSSION TITLE
International sorting

### DISCUSSION THREAD

**Discussion by kAlvaro** [symfony/symfony#58595] [Q&A] [answered]
Is there a builtin feature for Unicode and locale aware string sorting? I couldn't find it mentioned in the String component documentation and the search for "sort" didn't render related results.

I'm aware of PHP's `Collator::sort()`, I just want to know if there's a builtin wrapper or alternative.

**Comment by stof** [comment, marked as answer]
you should use the Collator from the intl extension for that. It is the right tool for the job. There is no reason for Symfony to reinvent its own implementation.

**Comment by kAlvaro** [comment]
Symfony already reimplements native features to make them easier or more coherent (`UnicodeString`), thus my question.
Thank you!
